scripts/test

```diff
@@ -29,6 +29,7 @@ ENTER_ROOT=$(pwd)/image/root ./bin/rio --debug agent -s https://localhost:7443 -
 
 export PATH=$(pwd)/bin:$PATH
 
+tail -f /var/lib/rancher/rio/agent/agent.log
```

Comment:
Did you want to background this? (`&`).  This will block the test script for ever.